main/templates/layouts/master.html

```diff
@@ -4,7 +4,7 @@
 <head>
     <meta charset='utf-8'>
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>{{ title }}{% block title %}{% endblock %}</title>
+    <title>{{ page_title }}</title>
```

Comment:
As a suggestion, you could have added here the `SarandONGa 💃 - ` so you dont have to add this to every page

Reply:
You are completely right 🤡
I will leave it the way it is now to save time, but in the future if any view is added, I will do this to refactor some code.
Thanks! 😊